test/functional/test_site/expected/sub_site/index.html

```diff
@@ -1,58 +1,64 @@
 <!DOCTYPE html>
 <html>
+
```

Comment:
The test site doesn't seem to have a page that uses our new headers.

Reply:
I avoided placing a Jumbotron on the test site as the current top nav bar will block it. 

After a PR is done to modify `<navbar>` in the vue-strap repo, we require another PR here to update our usage of `<navbar>`. Within that PR, we can add testing header elements.